### PR TITLE
LLS updates

### PR DESCRIPTION
**Pull request by profxj**
Ribaudo and more.

### CODE REVIEW

File: pyigm/surveys/llssurvey.py [hunk 1]
```diff
@@ -334,8 +335,40 @@ def load_HDLLS(cls, load_sys=True, grab_spectra=False, isys_path=None):
...
+        else:
+            lls = all_lls
+
+        # Dummy coords for now
+        lls['RA'] = np.arange(len(lls))
+        lls['DEC'] = np.arange(len(lls))
+        coords = SkyCoord(ra=lls['RA'], dec=lls['DEC'], unit='deg')
+
+        lls_survey = cls.from_sfits(lls, coords=coords)
+        lls_survey.ref = 'Ribaudo+13'
```
Comment (jsribaud) [inline pyigm/surveys/llssurvey.py]: Ribaudo+11

File: pyigm/surveys/tests/test_llssurvey.py [hunk 2]
```diff
@@ -34,6 +34,13 @@ def test_read_hdlls_dr1():   # This might actually be local now..
     assert len(gdCII) == 103
 
 
+def test_load_ribaudo13():
+    ribaudo13 = LLSSurvey.load_ribaudo()
```
Comment (jsribaud) [inline pyigm/surveys/tests/test_llssurvey.py]: ribaudo11

File: pyigm/surveys/tests/test_llssurvey.py [hunk 3]
```diff
@@ -34,6 +34,13 @@ def test_read_hdlls_dr1():   # This might actually be local now..
...
+    z, gz = ribaudo13.calculate_gz()
```
Comment (jsribaud) [inline pyigm/surveys/tests/test_llssurvey.py]: ribaudo11.calculate_gz()

File: pyigm/surveys/tests/test_llssurvey.py [hunk 4]
```diff
@@ -34,6 +34,13 @@ def test_read_hdlls_dr1():   # This might actually be local now..
     assert len(gdCII) == 103
 
 
+def test_load_ribaudo13():
```
Comment (jsribaud) [inline pyigm/surveys/tests/test_llssurvey.py]: test_load_ribaudo11()

**Review by jsribaud — COMMENTED** [review]
Suggest changing ribaudo13 everywhere to ribaudo11, although there is only one ribaudo survey, so it probably isn't confusing. Forgot about the erratum - maybe ribaudo13 makes more sense?

**Comment by profxj** [comment]
Whoops!  Sorry @jsribaud

**Comment by profxj** [comment]
Any other comments?

**Comment by jsribaud** [comment]
Sorry - had to take a break for a couple hours. I don't see anything else. Looks good.

File: pyigm/surveys/llssurvey.py (hunk 1 above)
Comment (profxj) [reply pyigm/surveys/llssurvey.py]: fixed

File: pyigm/surveys/tests/test_llssurvey.py (hunk 4 above)
Comment (profxj) [reply pyigm/surveys/tests/test_llssurvey.py]: all fixed

**Comment by profxj** [comment]
merging.

**Pull request merged**